ctpolicy/ctpolicy.go

```diff
@@ -76,3 +81,17 @@ func lifetimeInMonths(cert *x509.Certificate) int {
 	}
 	return lifetimeInMonths
 }
+
+// GroupByLogs reverses match-map between Logs and Groups. Returns map from log-URLs to set of Group-names.
```

Comment:
nit: for total clarity "Returns map from log-URLs to set of Group-names that contain the log"

Reply:
Done.